src/test/kotlin/com/atlassian/performance/tools/ssh/api/SshConnectionTest.kt

```diff
@@ -10,8 +14,47 @@ class SshConnectionTest {
         SshContainer().useConnection { ssh: SshConnection ->
             val sshResult = ssh.safeExecute("echo test")
 
-            Assert.assertTrue(sshResult.isSuccessful())
-            Assert.assertEquals(sshResult.output, "test\n")
+            assertThat(sshResult.isSuccessful()).isTrue()
+            assertThat(sshResult.output).isEqualTo("test\n")
         }
     }
+
+    @Test
+    fun shouldBeConcurrent() {
+        // given
+        val connectionConcurrency = 256
```

Comment:
@pbruski 40 parallel VU nodes should not be a problem for `newConnection`

Reply:
So this didn't detect https://github.com/atlassian/ssh/pull/7
Perhaps it's about concurrent connections to different hosts?